internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -1877,6 +1872,12 @@ func KexecPrepare(_ runtime.Sequence, data any) (runtime.TaskExecutionFunc, stri
 			}
 		}
 
+		if efi.GetSecureBoot() {
+			log.Print("kexec skipped as secure boot is enabled")
```

Comment:
kexec should have secure boot support iirc, yet this can be added later

Reply:
kexec works with secureboot, but that would break disk encryption aka trusted boot, also our UKI doesn't sign kernel if I'm right

Reply:
yeh, it's going to be a mess, since we also extend PCR from init, otherwise we need complex logic to see if we kexeced or not